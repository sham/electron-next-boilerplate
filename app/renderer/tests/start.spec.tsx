import * as React from 'react'
import * as Renderer from 'react-test-renderer'

import Start from '../pages/start'

test('page Start', () => {
  const component = Renderer.create(
    <Start />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
