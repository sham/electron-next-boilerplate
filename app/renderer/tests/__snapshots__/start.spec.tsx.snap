// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page Start 1`] = `
Array [
  <p
    className="jsx-600986976"
  >
    Electron + Next
  </p>,
  <a
    href="/caution"
    onClick={[Function]}
  >
    Caution
  </a>,
]
`;
